public/app/features/dashboard/utils/tracking.ts

```diff
@@ -0,0 +1,24 @@
+import { reportInteraction } from '@grafana/runtime';
+
+import { DashboardModel } from '../state';
+
+export function trackDashboardLoaded(dashboard: DashboardModel) {
+  // Count the different types of variables
+  const variables = dashboard.templating.list
+    .map((v) => v.type)
+    .reduce((r, k) => {
+      r[variableName(k)] = 1 + r[variableName(k)] || 1;
+      return r;
+    }, {});
+
+  reportInteraction('dashboard_loaded', {
```

Comment:
```suggestion
  reportInteraction('dashboards_init_dashboard_completed', {
```
In the Dashboards product, we namespace everything with `dashboards_` so we can filter out dashboard product events.